app/src/main/java/org/mozilla/focus/browser/InfoFragment.kt

```diff
@@ -46,6 +46,9 @@ class InfoFragment : EngineViewLifecycleFragment(), Session.Observer {
         // (e.g. wrong tracker count). However disabling tracking protection here shouldn't have any negative effects.
         engineSession.disableTrackingProtection()
 
+        // To avoid a visual glitch hide the WebView until the page is loaded.
+        webView!!.asView().visibility = View.GONE
```

Comment:
Nit: Probably effectively a non-issue, but it might be better to do this with `INVISIBLE` to avoid layout jitter.